index.html

```diff
@@ -3,8 +3,39 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
```

Comment:
[OPTIONAL]

- Good design implementation 👍🏽 . Please take note that your background color does not match the figma design. Please fix this so that your design can look pixel-perfect.

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227345092-48b5603d-50f4-4dbf-b337-c4820c025a47.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227345199-9da4a58e-e9a3-453f-b12c-1dac5ece929f.png)